static/faq.html

```diff
@@ -519,6 +519,11 @@ <h3><a href="#hardware-identifiers">Can apps access hardware identifiers?</a></h
                     around for compatibility. It used to need more extensive changes such as disallowing
                     access to the serial number but those restrictions are now standard.</p>
 
+                    <p>GrapheneOS adds the MediaDRM identifier permission which is revoked by default for
```

Comment:
It makes more sense to say disabled by default.

Reply:
This should probably also be turned into a usage guide section explaining that users may need to enable this to get DRM media working in an app.